packages/office-ui-fabric-react/src/components/SpinButton/docs/new/SpinButtonOverview.md

```diff
@@ -0,0 +1,5 @@
+A SpinButton allows someone to incrementally adjust a value in small steps. It’s mainly used for numeric values, but other values are supported too.
```

Comment:
Not Aneesha :) but unfortunately yes; it's one of the reasons the SpinButton code is so nasty today. The most common example is something like measurement units. (Personally I'd love to see the design changed so that the units are part of a static prefix/suffix like in TextField rather than inside the field itself.)